src/backend/distributed/shared_library_init.c

```diff
@@ -1895,7 +1895,7 @@ NodeConninfoGucCheckHook(char **newval, void **extra, GucSource source)
 		"sslcompression",
 		"sslcrl",
 		"sslmode",
-		"sslrootcert"
+		"sslrootcert",
 		"tcp_user_timeout",
```

Comment:
should the final comma be removed?

Reply:
both of them seems to be working, as in we can set the `tcp_user_timeout` variable with a comma at the end or without.

Reply:
Some older compilers might complain, but perhaps they've already gone extinct.